includes/class-rop.php

```diff
@@ -145,7 +145,7 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'admin_menu', $plugin_admin, 'menu_pages' );
 		$this->loader->add_action( 'rop_cron_job', $plugin_admin, 'rop_cron_job' );
 		$this->loader->add_action( 'rop_cron_job_once', $plugin_admin, 'rop_cron_job_once' );
-
+		$this->loader->add_action( 'init', $plugin_admin, 'check_cron_status', 20 );
```

Comment:
@mihaiiro any reason we used `init` here rather than `admin_init` ? If not then `admin_init` might be the better hook here so we're not constantly calling that function on website load

Reply:
The logic applied here is to make sure the Cron jobs will stop when required. 
If an admin presses the `Stop Sharing` button and then closes the admin area. If by any chance the Cron Jobs were not cleared/stopped, they will continue to run. 
Using `init` we make sure the Crons are cleared/stopped no matter what. 

The functions is checking the WP options and clears the Crons Jobs if it has to, this will not have an impact on user experience in front-end.